topostats/utils.py

```diff
@@ -124,13 +124,16 @@ def update_plotting_config(plotting_config: dict) -> dict:
         f"Main plotting options that need updating/adding to plotting dict :\n{pformat(main_config, indent=4)}"
     )
     for image, options in plotting_config["plot_dict"].items():
+        main_config_temp = main_config.copy()
         LOGGER.debug(f"Dictionary for image : {image}")
         LOGGER.debug(f"{pformat(options, indent=4)}")
         # First update options with values that exist in main_config
-        plotting_config["plot_dict"][image] = update_config(options, main_config)
+        if not plotting_config["plot_dict"][image]["core_set"]:  # avoids updating colurmap for diagnostic images
```

Comment:
Isn't the point of `update_plotting_config()` to allow people to update values though? :shrug: 

I can see it is possible that people might not get the results they expect if they don't correctly specify it so we need to be clear with the validation and documentation as to what is valid but we've tried to provide flexibility for people to change options rather than hard-coding values.